sr_utilities_common/src/sr_utilities_common/aws_manager.py

```diff
@@ -42,7 +43,19 @@ def __init__(self, access_key=None, secret_key=None, session_token=None):
             except IOError:
                 rospy.logerr("Could not find customer key, ask software team for help!")
             try:
-                response = requests.get('https://5vv2z6j3a7.execute-api.eu-west-2.amazonaws.com/prod', headers=headers)
+                for _ in range(retries):
+                    response = requests.get('https://5vv2z6j3a7.execute-api.eu-west-2.amazonaws.com/prod',
+                                            headers=headers)
+                    if response.status_code == 200:
+                        break
+                    else:
+                        rospy.logwarn(f"Response returned status code {response.status_code}, retrying... " +
+                                    f"(attempt {_ + 1}/{self._auth_retries})")
```

Comment:
should there be a slight delay between retries? SO as to not exceed the max request rate, whatever that may be.

Reply:
done